poster/styles/styles.scss

```
@@ -0,0 +1,296 @@
+    padding-left: 10px;
+    padding-top: 18px;
+}
+
+.info--2 {
+    background-color: #FF836F;
+    grid-column-start: 2;
+    grid-column-end: 2;
+    grid-row-start: 2;
+    grid-row-end: 2;
+    padding-left: 10px;
+    padding-top: 25px;
+}
+
+.info--3 {
+    background-color: #FF836F;
+    grid-column-start: 3;
+    grid-column-end: 3;
+    grid-row-start: 3;
+    grid-row-end: 3;
+    padding-left: 10px;
+    padding-top: 25px;
+}
+
+.info--4 {
+    background-color: #FF836F;
+    grid-column-start: 2;
+    grid-column-end: 2;
+    grid-row-start: 4;
+    grid-row-end: 4;
+    padding-left: 10px;
+    padding-top: 25px;
+}
+
+.info--5 {
+    background-color: #FF836F;
+    grid-column-start: 3;
+    grid-column-end: 3;
+    grid-row-start: 4;
+    grid-row-end: 4;
+    padding-left: 10px;
+    padding-top: 25px;
+}
+
+.info--6 {
+    background-color: #FF836F;
+    grid-column-start: 2;
+    grid-column-end: 2;
+    grid-row-start: 5;
+    grid-row-end: 5;
+    padding-left: 10px;
+    padding-top: 25px;
+}
+
+* {
+    transition-delay: 0s;
+    transition-duration: 0.5s;
+}
+
+/**MEDIA QUERY**/
+
+@media screen and (max-width: 1409px) {
+    .header--secondary {
+        font-size: 80px;
+    }
+}
+
+
+@media screen and (max-width: 1330px) {
+
+    .background-image {
+        top: 150px;
+    }
+
+    .info-secondary {
+        bottom: 30px;
+    }
+
+    .header {
+        font-size: 20px;
+    }
+
+    .header--secondary {
+        font-size: 70px;
+    }
+
+    .info {
+        grid-template-columns: 80px 80px 80px 80px;
+        grid-template-rows: 80px 80px 80px 80px 80px;
+        font-size: 14px;
+        padding-left: 5px;
+        padding-top: 15px;
+    }
+
+}
+
+@media screen and (max-width: 1010px) {
+    .info--1 {
+        grid-column-start: 1;
+        grid-column-end: span 2;
+        background-color: #FF836F;
+        text-transform: uppercase;
+        padding-left: 5px;
+        padding-top: 15px;
+    }
+
+    .info--2 {
+        grid-column-start: 1;
+        grid-column-end: 1;
+        padding-left: 5px;
+        padding-top: 22px;
+    }
+
+    .info--3 {
+        grid-column-start: 2;
+        grid-column-end: 2;
+        padding-left: 5px;
+        padding-top: 22px;
+    }
+
+    .info--4 {
+        grid-column-start: 1;
+        grid-column-end: 1;
+        padding-left: 5px;
+        padding-top: 22px;
+    }
+
+    .info--5 {
+        grid-column-start: 2;
+        grid-column-end: 2;
+        padding-left: 5px;
+        padding-top: 22px;
+    }
+
+    .info--6 {
+        background-color: #FF836F;
+        grid-column-start: 1;
+        grid-column-end: 1;
+        grid-row-start: 5;
+        grid-row-end: 5;
+        padding-left: 5px;
+        padding-top: 22px;
+    }
+
+}
+
+@media screen and (max-width: 870px) {
+    .background-image {
+        width: 450px;
+        top: 300px;
+        left: 195px;
+    }
+
+    .info-secondary {
+        left: 130px;
+        right: 0px;
```

Zero measurements do not need units.
```suggestion
        right: 0;
```